### PR TITLE
Add negative step response to fig 5.8

### PR DESCRIPTION
**Pull request by 92215302**

### CODE REVIEW

File: Figure_05_08.py [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
 #Figure 5.8
 G = (-s + 1)/(s + 1)
-Kc = [0.1, 0.5, 0.9]
+Kcs = [0.1, 0.5, 0.9]
 K = -1*s/((1 + 0.02*s)*(1 + 0.05*s))
 
 plt.figure('Figure 5.8')
-utilsplot.freq_step_response_plot(G, K, Kc, 0.2, 'S')
+utilsplot.freq_step_response_plot(G, K, Kcs, 0.2, 'S')
+plt.show()
```
Comment (alchemyst) [inline Figure_05_08.py]: by default, `plt.show()` will block, so this script will require you to close the first plot window to see the next one. It's probably better only to call `plt.show()` once at the end.

File: Figure_05_08.py [hunk 2]
```diff
@@ -1,15 +1,38 @@
...
+u1 = np.ones(500)
+u2 = np.zeros(500)
+u = np.hstack((u1, u2))
+
+plt.figure('Figure 5.8(b)')
+plt.title('(b) Response to step in reference')
+for Kc in Kcs:
+    K2 = Kc * K
+    T = feedback(G * K2, 1)
+    Ttf = lti(T.numerator, T.denominator)
```
Comment (alchemyst) [inline Figure_05_08.py]: It may be useful to wrap `lti.lsim` into the tf object like is already done for `step`.

**Comment by 92215302** [comment]
I may have misinterpreted the wrap comment.  Was your intent to modify utils to add a negative type step response to the tf object

**Comment by alchemyst** [comment]
My intent was to add a `tf.lsim()` method which works similarly to `tf.step()`.

**Pull request merged**